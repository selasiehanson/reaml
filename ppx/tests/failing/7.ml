let _foo =
 fun [@reaml.component] () ->
  let _ =
   fun () ->
    (* let[@reaml] () = () in *)
    ()
  in
  ()
