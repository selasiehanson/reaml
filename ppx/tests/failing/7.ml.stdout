[%%ocaml.error
  ;;"`@reaml.component` must be followed with a string - the component's name"
  ;;""]
